src/components/Contacts/ContactDetails/ContactDontationsTab/ContactDonationsTab.tsx

```diff
@@ -0,0 +1,123 @@
+import { Box, styled, Tab } from '@material-ui/core';
+import { Skeleton, TabContext, TabList, TabPanel } from '@material-ui/lab';
+import React from 'react';
+import { useTranslation } from 'react-i18next/';
+import { useGetContactDonationsQuery } from './ContactDonationsTab.generated';
+
+const ContactDonationsContainer = styled(Box)(({ theme }) => ({
+  padding: theme.spacing(0),
+}));
+
+const DonationsTabContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1),
+  background: theme.palette.background.paper,
+}));
+
+const DonationsGraphContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1),
+}));
+
+const DonationsTabList = styled(TabList)(({}) => ({
+  minHeight: 40,
+}));
+
+const DonationsTab = styled(Tab)(({ theme }) => ({
+  textTransform: 'none',
+  minWidth: 64,
+  minHeight: 40,
+  marginRight: theme.spacing(1),
+  color: theme.palette.text.primary,
+  opacity: 0.75,
+  '&:hover': { opacity: 1 },
+}));
+
+const ContactDonationsLoadingPlaceHolder = styled(Skeleton)(({ theme }) => ({
+  width: '100%',
+  height: '24px',
+  margin: theme.spacing(2, 0),
+}));
+
+interface ContactDontationsProp {
+  accountListId: string;
+  contactId: string;
+}
+
+enum DonationTabKey {
+  Donations = 'Donations',
+  PartnershipInfo = 'Partnership Info',
+}
+
+export const ContactDonationsTab: React.FC<ContactDontationsProp> = ({
+  accountListId,
+  contactId,
+}) => {
+  const { data, loading } = useGetContactDonationsQuery({
+    variables: { accountListId: accountListId, contactId: contactId },
+  });
```

Comment:
Do you think we should just load the graph data with this query and make the sub tabs load their own data? Maybe it's all about the same and makes sense to share.

Reply:
@OzzieOrca I was thinking this call would be for all of the data.  Since it is just one instance of contact.